packages/web-react/src/components/Box/demo/index.tsx

```diff
@@ -0,0 +1,28 @@
+// Because there is no `dist` directory during the CI run
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
```

Comment:
do we need these if we do not import icons?

Reply:
removed